<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Users</class>
 <widget class="QWidget" name="Users">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User managment</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources.qrc">
           <normaloff>:/images/images/add.png</normaloff>:/images/images/add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnDelete">
         <property name="text">
          <string>Delete</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources.qrc">
           <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QComboBox" name="cmbStatus"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnProperties">
         <property name="text">
          <string>Properties</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources.qrc">
           <normaloff>:/images/images/import.png</normaloff>:/images/images/import.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnPas">
         <property name="text">
          <string>Password</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources.qrc">
           <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="LAP_DataGrid" name="tbl"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LAP_DataGrid</class>
   <extends>QTableWidget</extends>
   <header location="global">lap_datagrid.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
